noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -470,29 +982,14 @@
                 ]]>
             </errors>
         </queries>
-    </rule>
-
-    <rule>
-        <title>Fonds count</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT
-                        a.systemid system_id,
-                        a.tittel title,
-                        (_ref_arkiv IS NULL) is_root,
-                        (COUNT(*) - 1) children_count
-                    FROM arkivstruktur.arkiv a
-                    LEFT JOIN arkivstruktur.arkiv b ON a._id = b._parent_id
-                    GROUP BY system_id, title, is_root
-                ]]>
-            </info>
-        </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST2">
         <title>Series count</title>
+        <description>
+            Provides information about the number of Series in the archive
```

Comment:
`Provides information about the number of series in the archive, and tests whether a fonds without any series exists`